syncfstriples/service.py

```diff
@@ -68,19 +68,22 @@ def load_graph(fpath: Path, format: str = None):
 def sync_removal(store: RDFStore, fname: str) -> None:
     ng = fname_to_ng(fname)
     store.drop_graph(ng)
-    store.forget_graph(ng)
+    store.forget_graph(ng) # TODO: discuss with MPO what the meaning of this is
```

Comment:
@cedricdcc best to check the docs in py-RDF-store it is explained over there

see:
- https://github.com/vliz-be-opsci/py-RDF-store/blob/main/pyrdfstore/store.py#L98
- https://github.com/vliz-be-opsci/py-RDF-store/blob/main/pyrdfstore/store.py#L115

basically - there is a functional difference between `drop` and `forget` which allows clients of the system to go for the one without the other -- this approach was taken because it feels like all combo strategies might be useful: drop-no-forget, forget-no-drop, both-drop-forget